src/main/webapp/javascript/user-page.js

```diff
@@ -73,33 +138,27 @@ setUpDivWithNoRecipes = (divID, message) => {
   recipesDiv.style.height = "100px";
 }
 
-// opens the recipe form modal
-function openModal() {
-  document.getElementById("recipe-modal").style.display = "block";
-  fetchBlobstoreUrl();
-}
-
 // not quite sure what this does; something with blobs
```

Comment:
If you have a better grasp of this method now that you've edited it some, add a more meaningful comment. If it's still some API or SO magic, it can stay.